qutebrowser/browser/commands.py

```diff
@@ -1847,13 +1663,23 @@ def debug_webaction(self, action, count=1):
             action: The action to execute, e.g. MoveToNextChar.
             count: How many times to repeat the action.
         """
-        member = getattr(QWebPage, action, None)
-        if not isinstance(member, QWebPage.WebAction):
+        tab = self._current_widget()
+
+        if tab.backend == tabmod.Backend.QtWebKit:
```

Comment:
Would it make sense to put this logic in the tab class, as this is (I think) the only point where the abstraction is leaking? Otherwise you'd need to repeat this on every call-site and e.g. plugin authors may forget to do this.